utils.py

```diff
@@ -31,12 +29,19 @@ def take_screenshot():
 
 
 def prepare_image(img):
+    import wx
+    wx.App()
```

Comment:
`record.py` is the only place that calls `take_screenshot` now and it already calls `wx.App()` since it boots a GUI. I think we can remove this call inside the `utils` module entirely since it was only moved here to support `play` in the first place. 

Instead of removing though why don't we pull this method back to the `record.py` file. What do you think?